app/tests/spec/models/unique-user-id.js

```diff
@@ -74,16 +74,5 @@ define(function (require, exports, module) {
       createUniqueUserId();
       assert.equal(uniqueUserId.get('uniqueUserId'), 'stored in uniqueUserId');
     });
-
-    it('migrates data stored in `localStorage.uuid` to `localStorage.uniqueUserId`', function () {
-      var storage = Storage.factory('localStorage', windowMock);
-      var uniqueUserId = 'originally stored in uuid';
-      storage.set('uuid', uniqueUserId);
-
-      createUniqueUserId();
-
-      assert.equal(storage.get('uniqueUserId'), uniqueUserId);
-      assert.isUndefined(storage.get('uuid'));
```

Comment:
@hritvi, instead of deleting the whole test, I'd probably just flip the two assertions, so that it does an `assert.isUndefined` for `uniqueUserId` and an `assert.equal` for `uuid`. Then tweak the description string to `does not migrate...`.

Not sure what anyone else thinks, but I generally prefer to assert a positive in the tests than use the absence of a test to imply behaviour.

Reply:
`storage.set('uuid', uniqueUserId)` sets uniqueUserId  some value. So it is failing to assert it as Undefined.

Reply:
Sorry, yeah, I didn't think that comment through! Really I just mean assert whatever the actual behaviour is, so for me this works when I try it locally:

```js
assert.equal(storage.get('uuid'), uniqueUserId);
assert.notEqual(storage.get('uuid'), storage.get('uniqueUserId'));
assert.ok(storage.get('uniqueUserId'));
```